docs/images/siddhi-logo.svg

```diff
@@ -0,0 +1,19 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please correct the licence header

Reply:
Fixed with a78d82e9a19fe778e1ecd773c02cc720aaa0eca2